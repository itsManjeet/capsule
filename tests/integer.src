println (1 + 1);
println (10);
println (10 - 1);
println (30 * 100);
println (40 / 10);
println (10 == 10);
println (20 == 10);
println (30 != 40);
println (40 != 48);
println (10 < 30);
println (10 <= 30);
println (10 >= 30);
println (10 > 30);
println (30 > 10);
println (30 >= 10);
println (10 >= 10);

let x = 10 * 20 / 30 + 30 * 39 / 29 + 58 - 10;
println (x);