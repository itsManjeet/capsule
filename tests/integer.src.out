2
10
9
3000
4
true
false
true
true
true
true
false
false
true
true
true
94
